### DISCUSSION TITLE
How to allocate resources for services when deploying in the k8s?

### DISCUSSION THREAD

**Discussion by GabrielWithTina** [vesoft-inc/nebula#5879] [Q&A]
Hi, 

We are evaluating the nebula in the AWS EKS. For the configuration of the production environment, I have read the following link:  https://docs.nebula-graph.io/3.6.0/4.deployment-and-installation/1.resource-preparations/#requirements_and_suggestions_for_installing_nebulagraph_in_production_environments

However, I could not find the resource allocation suggestion for each service (i.e. metad, graphd, storaged) when deploying in the k8s environment. Please help.

**Comment by wey-gu** [comment]
Maybe we need such doc chapters, too for k8s dementioning? @kevinliu24 @MegaByte875 @ChrisChen2023

**Reply by wey-gu** [reply]
docs issue created: https://github.com/vesoft-inc/nebula-docs/issues/2574

**Comment by pritish-moharir** [comment]
Hi @wey-gu we were also wondering the same, have there been any updates/developments here? Thanks!
